addons/mail/tests/test_mail_channel.py

```diff
@@ -431,15 +409,8 @@ def test_channel_unsubscribe_auto(self):
             "name": "Jonas",
         })
         test_partner = test_user.partner_id
-        test_channel_private = self.env['mail.channel'].with_context(self._test_context).create({
-            'name': 'Winden caves',
-            'description': 'Channel to travel through time',
-            'public': 'private',
-            'channel_partner_ids': [Command.link(self.user_employee.partner_id.id), Command.link(test_partner.id)],
-        })
```

Comment:
same remark (channel_type group instead?)

Reply:
There is group already in this test, so, we don't need another one to test